src/Aspire.Hosting.Dapr/IDistributedApplicationComponentBuilderExtensions.cs

```diff
@@ -30,7 +30,6 @@ public static class IDistributedApplicationResourceBuilderExtensions
     /// <param name="builder">The resource builder instance.</param>
     /// <param name="options">Options for configuring the Dapr sidecar, if any.</param>
     /// <returns>The resource builder instance.</returns>
-    [System.Diagnostics.CodeAnalysis.SuppressMessage("ApiDesign", "RS0027:API with optional parameter(s) should have the most parameters amongst its public overloads", Justification = "<Pending>")]
```

Comment:
Seems unrelated?

Reply:
Hmm not sure. Possibly I got a "suppression not required" diagnostic and acted on it? Will have to check

Reply:
Yep: 
![image](https://github.com/dotnet/aspire/assets/249088/5be82bef-273b-485e-b51c-fedfabc01f3f)